examples/espressif/esp/README.md

```diff
@@ -1,4 +1,20 @@
 # Examples for the Port `espressif-esp`
 
-- [Blinky](src/blinky.zig) on [ESP32-C3-32S-Kit](https://www.waveshare.com/wiki/ESP-C3-32S-Kit)  
+- [Blinky](src/blinky.zig) on [ESP32-C3-32S-Kit](https://www.waveshare.com/wiki/ESP-C3-32S-Kit)
   Showcases how to do a simple RGB cycling.
+
+## How to flash the image onto the device
+
+- esp image
+
+```sh
+esptool --chip esp32c3 -b 460800 --before default_reset --after hard_reset write_flash \
```

Comment:
nitpicking: Let's switch `-b` to `--baud` since we use the other long forms and I had to check to see what it meant.